docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+We will also show how to use the same reference curve to apply baseline correction for force signals in post processing.
+
+Positional Calibration
+----------------------
+
+The first step is to perform a positional calibration to calibrate the trap position to the distance measured by the camera.
+
+To do this calibration, we require a dataset acquired in the absence of a tether.
+In this dataset, trap 1 is moved over range, while the trap 1 position and camera-based distance is recorded.
+
+Note that this dataset should be acquired such that the bead tracking templates do not overlap.
+In case of overlap, make sure to slice the position and distance data such that this region is not included.
+Let's load this dataset and perform the distance calibration by invoking::
+
+    no_tether_data = lk.File("no_tether.h5")
+
+    distance_calibration = lk.DistanceCalibration(
+        calibration_data["Trap position"]["1X"], calibration_data.distance1, degree=2
+    )
+
+What this class does is regress a polynomial between the trap position and distance determined by the bead tracking.
+Note that this bead-to-bead distance typically already has the bead diameter subtracted, therefore it reflects the surface-to-surface distance.
+We can plot what this curve looks like by invoking `.plot()` on it::
+
+    distance_calibration.plot()
+
+.. image:: figures/piezotracking/mirror_calibration.png
+
+We can also inspect the residual, to determine how well the calibration model describes the data::
+
+    distance_calibration.plot_residual()
+
+.. image:: figures/piezotracking/mirror_calibration_residual.png
+
+Baseline correction
+-------------------
+
+Even without a tether, the measured force can be non-zero when the beads are close together.
+The next step is to characterize this baseline force so that we can subtract it from our force data.
+In principle, this step is optional, but it can improve the accuracy of your force-distance curves.
+We can quickly determine a polynomial baseline for both traps by invoking::
+
+    baseline_1x = lk.ForceBaseLine.polynomial_baseline(
+        no_tether_data['Trap position']['1X'], no_tether_data.force1x, degree=7, downsampling_factor=100
+    )
+    baseline_2x = lk.ForceBaseLine.polynomial_baseline(
+        no_tether_data['Trap position']['1X'], no_tether_data.force2x, degree=7, downsampling_factor=100
+    )
+
+Similarly as before, we can plot these baselines to verify that they describe the data well::
+
+    baseline_1x.plot(label="baseline f1x")
+    baseline_2x.plot(label="baseline f2x")
+    plt.legend()
+
+.. image:: figures/piezotracking/baseline.png
+
+And the residuals::
+
+    baseline_1x.plot_residual(label="baseline f1x")
+    baseline_2x.plot_residual(label="baseline f2x")
+    plt.legend(loc='lower right')
+
+.. image:: figures/piezotracking/baseline_residual.png
+
+The residuals should be distributed around zero and should ideally look like noise.
```

Comment:
I'm also wondering if `"should look like noise"` is too generic, but for now should be fine
```suggestion
The residuals should ideally look like noise distributed around zero.
```